examples/ai-billing/app/layout.tsx

```diff
@@ -0,0 +1,108 @@
+import { auth, signIn, signOut } from "@/auth";
+import { GeistMono } from "geist/font/mono";
+import { GeistSans } from "geist/font/sans";
+import type { Metadata } from "next";
+import { Toaster } from "sonner";
```

Comment:
lol yeah we're importing the toaster from sonner here, the file above looks like it can be deleted